README.adoc

```diff
@@ -1069,6 +1069,25 @@ User.all.size
 User.all.length
 ----
 
+=== Check if the collection is empty [[check-empty-collection]]
+
+When checking if the unloaded Active Record collection is empty, prefer `any?`/`empty?` over `present?`/`blank?`. The former executes a simple `EXISTS`-like query while the latter loads the whole collection to determine its size.
+
+[source,ruby]
+----
+# bad
+users.where(active: true).present?
+users.where(active: true).blank?
+
+# good
+users.where(active: true).any?
+users.where(active: true).empty?
+
+# good - the collection is used after loading
```

Comment:
I am referring to the "used" wording above.

Even though the collection will be used a few lines below, maybe only a fraction of it will be actually **used** with a batching iterator like `users.find_each { ...?

Or it will be further queried upon e.g. `users.active`? Or `users.last(3)`.

Or, it will be used in a sub-query, and it doesn't meet the purpose to load them all e.g. `Articles.recent.joins(:author).merge(users)`.